feat: :sparkles: responsibilities section (#91)
onlimit-study/dmp: documents/data-management-plan/responsibilities.qmd

# Responsibilities and Resources

Effective data management relies to a great extent on clear guidelines
as well as clear definitions of roles and responsibilities. By defining
responsibilities and ensuring the right expertise and tools are in
place, then data quality, compliance, and the long-term value of the

> project’s outputs should be ensured. This section outlines the key

Reviewer: ```suggestion
project's outputs should be ensured. This section outlines the key
```

Fixed that single quote character issue.

roles, and resources dedicated to managing data within the study.

## Responsibilities

This section outlines who is responsible for key aspects of data
handling (from collection and documentation to long-term storage and
sharing) and how collaboration between people and institutions supports

> the integrity and usability of the data throughout the study.

Reviewer: Does it need to describe how long these responsibilities last for and contingency plans for others who can take responsibility?

### Data Management Roles

> The project has appointed a dedicated Data Architect (DA) at Steno

Reviewer: In general, its good practice to limit the use of abbreviations, as it requires the reader to keep more "new terms" in their mind and track them throughout the document. "data architect" is simple enough to not need an abbreviation.

Diabetes Center Aarhus (SDCA) who holds primary responsibility for the
overall data management strategy. This includes overseeing data capture,
metadata production, data quality assurance, secure storage, backup and
recovery procedures, long-term archiving, and data sharing. The DA will
make sure that data flows are secure and well-documented.

### REDCap Setup and Study Team Contributions

While the DA is ultimately responsible for the setup and configuration

> of REDCap, much of the practical implementation will be carried out by a

Reviewer: ```suggestion
of REDCap, much of the practical implementation will be carried out by
```

> couple of members of the study team at Steno Diabetes Center Copenhagen

Reviewer: ```suggestion
some members of the study team at Steno Diabetes Center Copenhagen
```

(SDCC). These team members will also contribute to identifying and
describing specific data points registered in REDCap.

### Metadata and Documentation

> Metadata will be generated using [Seedcase

Reviewer: ```suggestion
Metadata will be organised using [Seedcase
```

Sprout](https://sprout.seedcase-project.org), based on the data recorded
in REDCap (for data registered there) and a data dictionary which will
be developed in collaboration with the LIVA team as the study progresses

> (it remains to be determined what data points we will get from them).

Reviewer: This seems like a comment? Should this be moved into a TODO or `content-hidden` block?

This will result in metadata that is consistent, comprehensive, and

> aligned with FAIR principles.

Reviewer: May need to have a glossary for all the terms.

Author: Done a link for now, but yes, something to keep in mind

### Long-term Data Handling and Storage

The DA is also responsible for overseeing the work carried out by the

> Seedcase team in converting raw data into parquet files for long-term

Reviewer: ```suggestion
Seedcase team in converting raw data into Parquet files for long-term
```

storage. It is essential that this process preserves the integrity of
the original data, and the DA will ensure that no information is lost or

> altered during the conversion. <!--TODO Check with Luke about this-->

Reviewer: I guess this section also is related to physical storage? Or?
